### ISSUE TITLE
Create a thumbnailer for .akira files

### ISSUE THREAD

**Issue by Philip-Scott** [akiraux/Akira#104]
## Expected Behavior
Akira files should show a thumbnail on file browsers of the available file. 

## Possible Solution
I once played with the idea of having Spice-Up show thumbnails of the file. I had it working on the Gnome File browser but it was not working on the elementary browser. But basically we just need to make the Application class have a cli arg that creates a thumbnail from the file. It's simpler if we can store a thumbnail inside the .akira zipped file while saving and just use that instead.

https://github.com/Philip-Scott/Spice-up/pull/121

**Comment by Alecaddd** [comment]
Yes, I agree that this is a pretty important aspect of the `.akira` file format.
Is this part of the work you're doing on #110 ?

**Comment by Philip-Scott** [comment]
Not on that PR for sure, but it will help to get it working :) That way we can store a thumbnail inside the zipped file instead of having to process the whole file when it's thumbnail is requested
